### ISSUE TITLE
'Couldn't connect to accessibility bus'

### ISSUE THREAD

**Issue by LinuxOnTheDesktop** [pdfpc/pdfpc#650]
**Expected Behaviour**

No error message upon launch.

**Actual Behaviour**

```
$ pdfpc s.pdf

(pdfpc:202944): dbind-WARNING **: 16:48:11.724: Couldn't connect to accessibility bus: Failed to connect to socket /root/.cache/at-spi/bus_0: Permission denied
Screensaver inhibited

[After quitting pdfpc:]
** (WebKitWebProcess:202980): WARNING **: 16:48:12.386: Can't connect to a11y bus: Could not connect: Permission denied
Screensaver reactivated
$
```

**Steps to reproduce**

1. Possibly necessary: be running Mint Mint 21 Cinnamon.
2. Possibly necessary: compile pdfpc from source. (There were some warnings, but I am afraid that I did not keep a record of them. I could re-create them.)
3. Run pdfpc, with a pdf file, from the terminal.

pdfpc version: compiled from the source that is here on github; I fetched that sort today.
Used Distribution (GTK version, vala version): Mint 21 Cinnamon.

```
$ dpkg -l libgtk* | grep -e '^i' | grep -e 'libgtk-*[0-9]'
ii  libgtk-3-0:amd64             3.24.33-1ubuntu2  amd64        GTK graphical user interface library
ii  libgtk-3-bin                 3.24.33-1ubuntu2  amd64        programs for the GTK graphical user interface library
ii  libgtk-3-common              3.24.33-1ubuntu2  all          common files for the GTK graphical user interface library
ii  libgtk-3-dev:amd64           3.24.33-1ubuntu2  amd64        development files for the GTK library
ii  libgtk-4-1:amd64             4.6.6+ds-0ubuntu1 amd64        GTK graphical user interface library
ii  libgtk-4-bin                 4.6.6+ds-0ubuntu1 amd64        programs for the GTK graphical user interface library
ii  libgtk-4-common              4.6.6+ds-0ubuntu1 all          common files for the GTK graphical user interface library
ii  libgtk2.0-0:amd64            2.24.33-2ubuntu2  amd64        GTK graphical user interface library - old version
ii  libgtk2.0-0:i386             2.24.33-2ubuntu2  i386         GTK graphical user interface library - old version
ii  libgtk2.0-bin                2.24.33-2ubuntu2  amd64        programs for the GTK graphical user interface library
ii  libgtk2.0-cil                2.12.40-3ubuntu1  amd64        CLI binding for the GTK+ toolkit 2.12
ii  libgtk2.0-common             2.24.33-2ubuntu2  all          common files for the GTK graphical user interface library
ii  libgtk3-perl                 0.038-1           all          Perl bindings for the GTK+ graphical user interface library
```

Display Server (X/Wayland): X.
HiDPI Setting (yes/no): yes.

**Further information**

pdfc did seem to run alright and whilst pdfpc was running the screensaver was inhibited (as it should be).

**Comment by fnevgeny** [comment]
> Failed to connect to socket /root/.cache/at-spi/bus_0: Permission denied

Running as root isn't a good idea anyway.

**Comment by LinuxOnTheDesktop** [comment]
Please note the '$' in `$ pdfpc s.pdf`: I was _not_ running pdfpc as root. Or at least I do not _think_ that I was. Please let me explain.

On _another_ PC, I do not see the error message when, as a normal user, I run pdfpc. And I find now that when I do the same thing, on the _original_ PC, everything works. (In both cases, I see this:

```
$ pdfpc test.pdf
Screensaver inhibited
[pdfpc runs and then terminates when I tell it to]
Screensaver reactivated
```
)

So perhaps unknowingly - and without reporting it - I _was_ originally running pdfpc as root. At any rate: some sort of warning about running as root - as against the cryptic error about permissions - _would be welcome_. Thank you for your time.

**Comment by fnevgeny** [comment]
> some sort of warning about running as root

It is not specific to pdfpc. You should not run *any* user-level app as the super user.

> as against the cryptic error about permissions

The "cryptic error" is provided by your OS.

**Comment by LinuxOnTheDesktop** [comment]
When run as root pdfpc would do well to say: you may encounter problems or see warnings. Many an application issues a warning of such a useful sort. Amending pdfpc to say such a thing would be trivial for you. (Not for me; I know no vala.)
